src/main/java/org/example/Kingdom/Main.java

```diff
@@ -178,7 +158,13 @@ else if (pastType == 4) {
             game.beforeDay1_2();
         }
 
-        for (int i=0; i<5;) {
+        sleep250_5times();
+
+        game.day1_1();
+    } // 당신은 전생에 어떤 사람이었습니까?
+
+    public void sleep250_5times() {
+        for (int i = 0; i < 5; ) {
```

Comment:
```suggestion
    public void sleep(int duration, int times) {
        for (int i = 0; i < times; ) {
```

위 처럼 변경하면 호출 할때

sleep250_5times() 이 아니고

sleep(250, 5); 이런식으로 조금 더 유연하게 코드를 만들 수 있어요.

Reply:
굳이 반복문을 유지하는것도 불필요해보이네요.